### PR TITLE
Feature/coveralls

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,26 +20,33 @@ install:
...
-  - conda config --add channels conda-forge
-  - conda config --add channels bioconda
   
 ## Download cpan libraries
-## # For some reason DB_File installation fails, it is not needed for the test
+## For some reason DB_File installation fails, it is not needed for the test
   - sed -i '/DB_File/d' ./definitions/cpanfile
   - cd definitions; cpanm --quiet --notest --installdeps .
   - cd -
+## Set up for test
```
Comment (henrikstranneheim) [inline .travis.yml]: Set up for test -> Set up for cpanm dependencies test

File: .travis.yml [hunk 2]
```diff
@@ -20,26 +20,33 @@ install:
...
+  - cpanm --notest Devel::Cover::Report::Coveralls
 
-script:
-## Run MIP install script for the rare disease piplines's main environment programs 
-  - perl t/mip_install.test
+install: 
+## Test installation script
+  - PERL5OPT=-MDevel::Cover=-coverage,statement,branch,condition,path,subroutine perl t/mip_install.test
```
Comment (henrikstranneheim) [inline .travis.yml]: Test installation script -> Test coverage for installation script

File: .travis.yml [hunk 3]
```diff
@@ -20,26 +20,33 @@ install:
...
+
+script:
+## Test MIP
   - source activate mip_travis
   - cd /lib/x86_64-linux-gnu
   - sudo ln -s "$ROOTSYS"/lib/*so .
   - cd -
-  - prove t -r
-  - perl t/mip_analyse_rare_disease.test
+  - PERL5OPT=-MDevel::Cover=-coverage,statement,branch,condition,path,subroutine prove -lr t
```
Comment (henrikstranneheim) [inline .travis.yml]: '##' Set-up test coverage for test directory "t" and "analyse rare disease pipeline"

File: .travis.yml (hunk 2 above)
Comment (jemten) [reply .travis.yml]: This both launches the mip_analysis.test script and gathers coverage information from that script.

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**